packages/module/release.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  branches: [ { name: 'main', channel: 'prerelease' } ],
+  branches: [ { name: 'main', channel: 'prerelease' }, { name: 'v5', channel: 'alpha', prerelease: 'alpha' } ],
```

Comment:
Same question about prerelease vs alpha for this.

Also are we wanting `main` to be the v5 branch, or do we want a v5 branch here? In most of the other extensions we've moved to having `main` as v5 and a `v4` branch for v4.